### ISSUE TITLE
error about squeeze

### ISSUE THREAD

**Issue by cxmscb** [dongyp13/Translation-Invariant-Attacks#2]
'''
InvalidArgumentError (see above for traceback): Tried to explicitly squeeze dimension 1 but dimension was not 1: 2
'''

**Comment by dongyp13** [comment]
Sorry, where does the error occur?

**Comment by tuji-sjp** [comment]
> '''
> InvalidArgumentError (see above for traceback): Tried to explicitly squeeze dimension 1 but dimension was not 1: 2
> '''

This could be a version issue for python and TensorFlow.
You can try to change it with the README.

**Comment by lisenxd** [comment]
I met the same issue. Did you solve it?

**Comment by lisenxd** [comment]
> Sorry, where does the error occur?
Here:
    with slim.arg_scope(inception_v3.inception_v3_arg_scope()):
        logits_v3, end_points_v3 = inception_v3.inception_v3(
            input_diversity(x), num_classes=num_classes, is_training=False)

**Comment by YuehuaZhu** [comment]
I met the same issue

**Comment by lisenxd** [comment]
> I met the same issue. Did you solve it?

solved

**Comment by Ichbinhippo** [comment]
Hi,Yinpeng. a question. why do you set the prediction of inception_resnet_v2 as the target label? and the loss is computed by the (y,pred_logits). so is this means that you want to genarate a adv with target label as inception_resnet_v2 predicted rather than a adv that can fool all the 4 models you gave in the model? if it is convient, hope to get you reply? thx
